OpenRA.Game/Widgets/WorldInteractionControllerWidget.cs

```diff
@@ -305,8 +305,7 @@ static IEnumerable<Actor> SelectActorsInBoxWithDeadzone(World world, int2 a, int
 			return world.ScreenMap.ActorsInBox(a, b)
 				.Where(x =>
 				{
-					var s = x.TraitOrDefault<Selectable>();
-					return s != null && s.Info.Selectable && (x.Owner.IsAlliedWith(world.RenderPlayer) || !world.FogObscures(x));
+					return x.HasTrait<Selectable>() && (x.Owner.IsAlliedWith(world.RenderPlayer) || !world.FogObscures(x));
```

Comment:
This can now be further simplified to
`x => x.HasTrait<Selectable>() && (x.Owner.IsAlliedWith(world.RenderPlayer) || !world.FogObscures(x))`